grid_map_core/CMakeLists.txt

```diff
@@ -127,6 +129,7 @@ endif()
 
 ament_export_targets(export_${PROJECT_NAME} HAS_LIBRARY_TARGET)
 ament_export_dependencies(Eigen3)
-ament_package(CONFIG_EXTRAS
-  cmake/${PROJECT_NAME}-extras.cmake
-)
+ament_package()
+#ament_package(CONFIG_EXTRAS
+#  cmake/${PROJECT_NAME}-extras.cmake
```

Comment:
This breaks if we leave it in - any tips on how to test the plugins are working in another package?